js/shop.js

```diff
@@ -0,0 +1,219 @@
+// allowed_terms is a whitelist of Canvas enrollment_term_ids where shopping is allowed
+// NOTE - the term ids in allowed_terms must be strings, not ints
+var allowed_terms = ["69"];
```

Comment:
Have we verified if this the right term for colgsas? I see 69 for both hds and colgsas. 

In the previous setup,  I think we had discussed checking shop.js with blank allowed_terms and devops would populate the appropriate term during deployment for each semester. I am ok with a value but we may not know the correct term ids..